src/components/SearchSection/ScoreReactionBoard/ScoreReactionTable/ScoreReactionTable.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+import styles from './ScoreReactionTable.module.css';
+import SRTableRows from './SRTableRows';
+
+const ScoreReactionTable = ({ scoreReactionBoardList }) => {
+    return (
+        <div className={styles.scoreReactionTable}>
+            <table className={styles.table}>
```

Comment:
Here thead and tbody tags are missing
![image](https://github.com/user-attachments/assets/8bd15bcf-3940-447f-bcca-d46dafaa7f71)